tools/bcl-test-importer/BCLTestImporter/BCLTestProjectGenerator.cs

```diff
@@ -437,8 +438,15 @@ internal static IEnumerable<string> GetIgnoreFiles (string templatePath, string
 			}
 			// do we have ignores per files and not the project name? Add them
 			foreach (var (assembly, hintPath) in assemblies) {
-				foreach (var platformFile in GetIgnoreFileNames (assembly, platform)) {
-					var commonAssemblyIgnore = Path.Combine (templateDir, GetCommonIgnoreFileName (assembly, platform));
+				// we could be looking at a splitted assembly, if that is the case, less pass the name of the dll without the 'part{number}.dll
+				// so that we have all the ignores in a single file
+				var assemblyName = assembly;
+				if (assembly.Contains (splitPattern)) {
+					var index = assembly.IndexOf(splitPattern, StringComparison.Ordinal);
```

Comment:
minor: space before `(`

Reply:
that can be simplified, just to the `IndexOf`
i.e. there's no need to call `Contains` first, `IndexOf` will return `-1` if the content is not present